fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -37,7 +37,7 @@ Usage() {
   echo "             --SEPhaseNeg=<input spin echo negative phase encoding image>"
   echo "             --SEPhasePos=<input spin echo positive phase encoding image>"
   echo "             --echospacing=<effective echo spacing of fMRI image, in seconds>"
-  echo "             --unwarpdir=<unwarping direction: x/y/z/-x/-y/-z>"
+  echo "             --unwarpdir=<PE direction for unwarping according to the *voxel* axes: {x,y,x-,y-} or {i,j,i-,j-}>"
```

Comment:
Is there a reason why this no longer mentions z/z- and k/k- ?

Reply:
Because, in principle for 2D EPI data, at least when converted with 'dcm2niix', the unwarpdir will never be the z/k axis.  That's why I removed z/k as listed options from the --unwarpdir argument in TopPreprocessingAll.sh as well.  In the case of TopPreprocessingAll.sh, z/k is explicitly not a supported option, and I just added an error in that script if --unwarpdir is not a supported value.

In the case of DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh, there is no error check on the allowed values for --unwarpdir currently.  I'll do whatever others think best.  (i.e., either change the usage back to include z/k, add an explicit check for allowed values, or just leave as is).

Reply:
I would always vote for an explicit check for allowed values.

Reply:
Easy enough to add an explicit check.  I just need @glasserm and @coalsont thoughts on whether z/k should be an allowable option or not in DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh.  I guess we could nominally allow z/k within that particular script (consistent with how it was), although note that if using SEFMs as the distortion correction approach, the use of z/k will get kicked out as an error once TopPreprocessingAll.sh is called.

Reply:
I've left z/k as allowed values in this specific script, because presumably, under the path that uses gradient echo field maps (epi_reg), that is still an allowed PE dir.